app/views/news_items/show.html.erb

```diff
@@ -1,4 +1,5 @@
-<%= content_for :head, news_feed_link_tag %>
+<% content_for :head, news_feed_link_tag %>
+<% content_for :title, "#{@news_item.title} - #{t('news_item.title.'+"#{@news_item.service_name}")}" %>
```

Comment:
I think there might be a helper for this

Reply:
Which part? I should have added the content for head i dont think was working properly before

Reply:
I tried looking for any helpers but this is closest i could fine.